### PR TITLE
Fix the keyboard shortcut for opening file search

### PR DESCRIPTION
**Pull request by jaril**
This gets us a natural Cmd+Shift+F keyboard shortcut to pop open full text search, regardless of which view (devtools/viewer) you're in.

### CODE REVIEW

**Comment by jasonLaster** [comment]
One quick bug, but otherwise good.

CMD+Shift+F in non-dev should open devtools

**Comment by jaril** [comment]
> One quick bug, but otherwise good.

Good catch — fixing.

File: src/ui/components/KeyboardShortcuts.tsx [hunk 1]
```diff
@@ -15,11 +15,19 @@ function setupShortcuts() {
 
 const globalShortcuts = setupShortcuts();
 
-function KeyboardShortcuts({ setSelectedPrimaryPanel }: PropsFromRedux) {
+function KeyboardShortcuts({ viewMode, setSelectedPrimaryPanel, setViewMode }: PropsFromRedux) {
+  const openFileSearch = () => {
```
Comment (jasonLaster) [inline src/ui/components/KeyboardShortcuts.tsx:19]: i think we should refer to this as `FullTextSearch` or ProjectSearch

File: src/devtools/client/debugger/src/components/ProjectSearch.js [hunk 2]
```diff
@@ -199,7 +199,7 @@ export class ProjectSearch extends Component {
   };
 
   onEnterPress = () => {
-    if (!this.isProjectSearchEnabled() || !this.state.focusedItem || this.state.inputFocused) {
+    if (!this.state.focusedItem || this.state.inputFocused) {
```
Comment (jasonLaster) [inline src/devtools/client/debugger/src/components/ProjectSearch.js:202]: nice

**Review by jasonLaster — APPROVED** [review]

**Pull request merged**